src/components/AttachmentModal.js

```diff
@@ -12,7 +12,6 @@ import AttachmentView from './AttachmentView';
 import AttachmentCarousel from './AttachmentCarousel';
 import styles from '../styles/styles';
 import * as StyleUtils from '../styles/StyleUtils';
-import * as FileUtils from '../libs/fileDownload/FileUtils';
```

Comment:
`FileUtils` cannot be removed as it is being used in other places as well.

Reply:
Having issues with eslint on my editor and missed this. Thanks!